metabase/metabase
# Fix back to the dashboard button with x-ray dashboards

Epic https://github.com/metabase/metabase/issues/29500

How to test:
- Browse -> Sample Database -> Hover on Orders and click X-ray this
- Click on card title in the x-ray dashboard
- There should be a back button available
- Make sure you can navigate back and it works

<img width="1248" alt="Screenshot 2023-05-24 at 15 36 55" src="https://github.com/metabase/metabase/assets/8542534/c5ad8d76-a9b8-433e-ab5f-30dad41a7d7d">
<img width="1405" alt="Screenshot 2023-05-24 at 15 37 29" src="https://github.com/metabase/metabase/assets/8542534/3a80ddb2-4c17-4282-a7f4-112d4eeef0ec">
<img width="1261" alt="Screenshot 2023-05-24 at 15 37 38" src="https://github.com/metabase/metabase/assets/8542534/941f7c4e-27e4-451e-8ce7-dff0d3a48c9d">
